test-app/tests/unit/changeset-test.js

```diff
@@ -3099,6 +3099,124 @@ module('Unit | Utility | changeset', function (hooks) {
     assert.ok(get(dummyChangeset, 'isValid'), 'should be valid');
   });
 
+  /**
+   * #removeError
+   */
+
+  test('#removeError removes an error from the changeset', async function (assert) {
```

Comment:
@jkeen could you please either add new test or add an assertion to this one to that checks that field values does not get reset? I assume it’s an important aspect of this new method

Reply:
Great point, added a new test to verify that @SergeAstapov